sample-apps/donet-sample-app/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.7"
+services:
+  otel:
+    image: amazon/aws-otel-collector:latest
+    command: --config /config/collector-config-local.yml
+    volumes:
+      - ~/.aws:/root/.aws:ro
+      - .:/config
```

Comment:
Why is this copying everything into `/config`?

Reply:
I tried to revert back on using a `./collector-config-local.yml` directory but that does not seem to work. As there is no spec that specifies the dir structure, I decided to keep it at `/config`

Reply:
```suggestion
      - ./collector-config-local.yml:/config/collector-config-local.yml
```
Does this work?